contracts/sales/FairSale.sol

```diff
@@ -18,9 +17,19 @@ contract FairSale {
     using IterableOrderedOrderSet for bytes32;
     using IdToAddressBiMap for IdToAddressBiMap.Data;
 
+    modifier notInitialized() {
+        require(!initialized, "already initialized");
+        _;
+    }
+
+    modifier onlyDeployer {
+        require(msg.sender == deployer, "FixedPriceSale: FORBIDDEN");
+        _;
+    }
+
     modifier atStageOrderPlacement() {
         require(
-            block.timestamp < endDate,
+            block.timestamp < auctionEndDate,
```

Comment:
```suggestion
            block.timestamp < saleEndDate,
```

Reply:
We're dropping `sale`, and `auction` since the scope/context is `<Type>Sale` contract